templates/components/itilobject/timeline/form_task.html.twig

```diff
@@ -69,7 +69,11 @@
             {% if entry_i['groups_id_tech'] %}
                <span class="badge bg-orange-lt">
                   <i class="fas fa-users me-1"></i>
-                  {{ get_item_link('Group', entry_i['groups_id_tech'], {'enable_anonymization': true}) }}
+                  {% if get_current_interface() == 'helpdesk' %}
+                     {{ get_item_name('Group', entry_i['groups_id_tech'], {'enable_anonymization': true}) }}
```

Comment:
I may be wrong, but I think that `get_item_name` will not correctly handle the anonymization. The `get_item_link` method is supposed to return the name when the user does not have right to access the item. If it does not, it probablty has to be fixed.

Reply:
IMHO, `Group::getAnonymizedName()` should return null instead of an empty string when anonymization is disabled, and checks done on this returned value should be updated.